int16 debug_num
int16[100] sensor_data_x
int16[100] sensor_data_y
